source/dotnet/Test/AdaptiveCards.Test/AdaptiveCardApiTests.cs

```diff
@@ -814,5 +814,157 @@ public void BackgroundImageBackCompatWithLegacyUriType()
+            
+            var container = (AdaptiveContainer)columns[1].Items[0];
+            Assert.AreEqual(300u, container.PixelMinHeight);
+        }
+
+        [TestMethod]
+        public void TestObjectModelMinHeight()
+        {
+            AdaptiveCard card = new AdaptiveCard(new AdaptiveSchemaVersion("1.2"))
+            {
+                PixelMinHeight = 500,
+                Body = new List<AdaptiveElement>
+                {
+                    new AdaptiveColumnSet()
+                    {
+                        PixelMinHeight = 100,
+                        Columns = new List<AdaptiveColumn>
+                        {
+                            new AdaptiveColumn
+                            {
+                                PixelMinHeight = 200,
+                                Items = new List<AdaptiveElement>
+                                {
+                                    new AdaptiveTextBlock
+                                    {
+                                        Text = "TextBlock"
+                                    }
+                                }
+                            },
+                            new AdaptiveColumn
+                            {
+                               Items = new List<AdaptiveElement>
+                               {
+                                   new AdaptiveContainer
+                                   {
+                                       PixelMinHeight = 300,
+                                       Items = new List<AdaptiveElement>
+                                       {
+                                            new AdaptiveTextBlock
+                                            {
+                                                Text = "TextBlock"
+                                            }
+                                       }
+                                   }
+                               }
+                            }
+                        }
+                    }
+                }
+            };
+            
+            var expectedJson =
+@"{
+  ""type"": ""AdaptiveCard"",
+  ""version"": ""1.2"",
+  ""body"": [
+    {
+      ""type"": ""ColumnSet"",
+      ""columns"": [
+        {
+          ""type"": ""Column"",
+          ""items"": [
+            {
+              ""type"": ""TextBlock"",
+              ""text"": ""TextBlock""
+            }
+          ],
+          ""minHeight"": ""200px""
+        },
+        {
+          ""type"": ""Column"",
+          ""items"": [
+            {
+              ""type"": ""Container"",
+              ""items"": [
+                {
+                  ""type"": ""TextBlock"",
+                  ""text"": ""TextBlock""
+                }
+              ],
+              ""minHeight"": ""300px""
+            }
+          ]
+        }
+      ],
+      ""minHeight"": ""100px""
+    }
+  ],
+  ""minHeight"": ""500px""
+}";
```

Comment:
[nit] Fix the indentation

Reply:
I see, this is because we're comparing this to what comes out of ToJson. Could you add a comment to that effect?

---
In reply to: [274046694](https://github.com/Microsoft/AdaptiveCards/pull/2638#discussion_r274046694) [](ancestors = 274046694)